torchbenchmark/models/pytorch_CycleGAN_and_pix2pix/__init__.py

```diff
@@ -34,6 +36,11 @@ def __init__(self, device='cpu', jit=False):
     def get_module(self):
         return self.model, self.input
         
+    def set_train(self):
```

Comment:
isn't it better to override set_mode?  if someone calls set_mode or set_eval it'd still call through to self.model.eval() i guess?  although i think it's also true that the overriding is not technically important here, since if people use the APIs correctly they will call set_eval before using the eval model, and the train model will always work right...